tests/apollo/util/skvbc.py

```diff
@@ -22,6 +23,7 @@
 
 WriteReply = namedtuple('WriteReply', ['success', 'last_block_id'])
 
+CLIENT_REQ_GRACEFUL_END_TIMER=3
```

Comment:
client requests timeout by default, if not changed can be taken from bft.py REQ_TIMEOUT_MILLI which is 5 seconds. you can assign this variable from there or just remove it and import REQ_TIMEOUT_MILLI from bft. I would add 1 more second to be on the safe side (so actual value will be 6):

```
from bft import REQ_TIMEOUT_MILLI
....
....
CLIENT_REQ_GRACEFUL_END_TIMER = REQ_TIMEOUT_MILLI + 1

```